src/app/screens/Calculator/index.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react';
+import {View, TextInput} from 'react-native';
+import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view';
+
+import styles from './styles';
+import Button from './components/Button';
+import { retrieveButtons } from './utils';
+
+const Calculator = () => {
+  //Variables y funcionesa
+  const [display, setDisplay] = useState('');
+
+  const leerPresionado = target => {
+    GET_BUTTONS.find(button => button.label === target).action();
+  };
+
+  let GET_BUTTONS = retrieveButtons(display, setDisplay);
```

Comment:
This is const, since you are not modifying this. Also UPPER_CAMEL_CASE should only be used for constants outside the component.

Reply:
also, should say buttons and not get buttons

Reply:
Understood, make the pertinent modifications.